userspace/libsinsp/test/scap_files/converter_tests.cpp

```diff
@@ -503,3 +503,52 @@ TEST_F(scap_file_test, shutdown_x_check_final_converted_event) {
 	uint8_t how = 1;
 	assert_event_presence(create_safe_scap_event(ts, tid, PPME_SOCKET_SHUTDOWN_X, 3, res, fd, how));
 }
+
+////////////////////////////
+// PTRACE
+////////////////////////////
+
+TEST_F(scap_file_test, ptrace_e_same_number_of_events) {
+	open_filename("ptrace.scap");
+	assert_num_event_type(PPME_SYSCALL_PTRACE_E, 3);
+}
+
+TEST_F(scap_file_test, ptrace_x_same_number_of_events) {
+	open_filename("ptrace.scap");
+	assert_num_event_type(PPME_SYSCALL_PTRACE_X, 3);
+}
+
+TEST_F(scap_file_test, ptrace_x_check_final_converted_event) {
+	open_filename("ptrace.scap");
+
+	// Inside the scap-file the event `463` is the following:
+	// - type=PPME_SYSCALL_PTRACE_X,
+	// - ts=1747834548577695347
+	// - tid=368860
+	// - args=res=0 addr=78 data=3B
+	//
+	// And its corresponding enter event `464` is the following:
+	// - type=PPME_SYSCALL_PTRACE_E
+	// - ts=1747834548577692897
+	// - tid=368860
+	// - args=request=4(PTRACE_PEEKUSR) pid=368861(mystrace)
+	//
+	// Let's see the new PPME_SYSCALL_SETUID_X event!
```

Comment:
This should obviously say:

```suggestion
	// Let's see the new PPME_SYSCALL_PTRACE_X event!
```
but seems like an overkill to trigger a new test run. I can do the fix if there will be need to run rebase again before merge, or I'll fix it in the next PR.